### PR TITLE
feat(dashboard-pdf-export): connect pdf overlay to dashboardDetailPage

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
image

First of all, I only connected the pdfExport button.
the preview component has been created and additional application is required after configuring the detail page later.

### Things to Talk About
I'm thinking about how to configure widget placement for pdf..

### CODE REVIEW

File: src/services/dashboards/dashboard-detail/modules/DashboardDetailPreview.vue [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+import type { DashboardLayoutWidgetInfo } from '@/services/dashboards/widgets/config';
+
+
+interface Props {
+    dashboardId: string;
+}
+const props = defineProps<Props>();
+
+const dashboardRef = ref<HTMLElement|null>(null);
+const dashboardTitleRef = ref<Vue|null>(null);
```
Comment (minseolee) [inline src/services/dashboards/dashboard-detail/modules/DashboardDetailPreview.vue:53]: 👍👍

File: src/services/dashboards/dashboard-detail/modules/DashboardDetailPreview.vue [hunk 2]
```diff
@@ -0,0 +1,133 @@
+<template>
+    <div ref="dashboardRef"
+         class="dashboard-detail-preview"
```
Comment (minseolee) [inline src/services/dashboards/dashboard-detail/modules/DashboardDetailPreview.vue:3]: It looks like `DashboardDetailPreview.vue` looks very similar with `DashboardDetailPage.vue`.
If there is some code changes on DetailPage, Should the same codes be written twice?

**Review by minseolee — COMMENTED** [review]
I left little comments :)

File: src/services/dashboards/dashboard-detail/modules/DashboardDetailPreview.vue (hunk 2 above)
Comment (sulmoJ) [reply src/services/dashboards/dashboard-detail/modules/DashboardDetailPreview.vue:3]: That's right. If it's a code that should be displayed on the pdf, it should be written. 😢

**Review by song2kim — APPROVED** [review]
LGTM~

**Review by minseolee — APPROVED** [review]

**Pull request merged**